data-import/desktop-csv-import/desktop-csv-import.adoc

```diff
@@ -0,0 +1,420 @@
+When you do that, you will see two buttons appear for creating a local graph or connecting to a remote graph.
+
+image::{img}desktop_add_graph.jpg[link="{img}desktop_add_graph.jpg",role="popup-link"]
+
+Choose the `Create a Local Graph` button, and fill in some details on the entry form for your database.
+You can leave whatever version is defaulted in the last dropdown.
+Once that is complete, click `Create`.
+
+image::{img}desktop_db_details.jpg[link="{img}desktop_db_details.jpg",role="popup-link"]
+
+It may take a minute or two to create the instance, but once it finishes, you should see something like the screenshot below.
+You can start your database by clicking the `Start` button on it.
+
+image::{img}desktop_db_created.jpg[link="{img}desktop_db_created.jpg",role="popup-link"]
+
+When it starts, the button should change to `Stop` so you now see the screen below.
+
+image::{img}desktop_start_db.jpg[link="{img}desktop_start_db.jpg",role="popup-link"]
+
+The other button is to manage the database.
+Let's click on `Manage` to show a screen like this one.
+
+image::{img}desktop_manage_db.jpg[link="{img}desktop_manage_db.jpg",role="popup-link"]
+
+The top panel in the image above shows some basic functions you can execute on the database.
+The next step will show us how to use this panel to put our files in the `Import` folder.
+
+=== 2. Putting CSV files in the expected folder
+
+First, download this https://s3.amazonaws.com/dev.assets.neo4j.com/wp-content/uploads/desktop-csv-import.zip[zip file^] containing 3 CSV files for products, orders, and order details.
+We will need to put these files in a folder that Neo4j expects for imports.
+
+On the Mac, open a Finder window and navigate to where the zip file was downloaded.
+We will come back to this location in a minute.
+
+In Neo4j Desktop's `Manage` pane, you should see the button for `Open Folder` in the upper panel.
+Right next to that button is a dropdown arrow button.
+If you click that, you will see a menu of options for different folders.
+
+image::{img}desktop_open_folder.jpg[link="{img}desktop_open_folder.jpg",role="popup-link"]
+
+Choose the `Import` folder.
+This will open up another Finder window on the Mac.
+//TODO: add in Windows version here!!!
+
+image::{img}desktop_import_folder.jpg[link="{img}desktop_import_folder.jpg",role="popup-link"]
+
+Now you can click on the other Finder window where the zip file downloaded and move the entire .zip file to the folder Neo4j Desktop opened.
+You should see something like this when you are finished.
+
+image::{img}desktop_folder_with_zip.jpg[link="{img}desktop_folder_with_zip.jpg",role="popup-link"]
+
+Double-click on the zip file, and it should unzip and create a `desktop-csv-import` folder.
+If you click on that folder, you should see 3 .csv files - products.csv, orders.csv, and order-details.csv.
+You can open each of the files to see what the data looks like.
```

Comment:
show example like first two or three lines.

Reply:
Do you think that is needed when we look at it with Cypher in a few paragraphs? I'm not sure showing the files here is necessary when we review them with Cypher later (and, depending on the program they use to open the files, they could look completely different).